### ISSUE TITLE
Storyboard 

### ISSUE THREAD

**Issue by wghuneke** [adele157/easterlies-collaborative-project#37]
Suggested figures that go into paper. Add example figure, short description, suggestions for panels/what needs to be changed...

**Comment by wghuneke** [comment]
**Figure 1: Experimental design (enhancing/relaxation of climatological wind field)**
a) climatological wind field in JRA
b) historical trends in JRA and CMIP
c) projected trends in CMIP
d) applied perturbation

**Comment by adele157** [comment]
I've also added a [figure summary page here](https://github.com/adele157/easterlies-collaborative-project/blob/master/Figure_outline.md). I suggest we keep the discussion here in this issue, but can put the result of this discussion on that figure summary page, to keep a cleaner version and somewhere to have the figures in order.

**Comment by wghuneke** [comment]
Sounds good. Thought it's easier to add figures in an issue.

**Comment by julia-neme** [comment]
> **Figure 1: Experimental design (enhancing/relaxation of climatological wind field)**
> a) climatological wind field in JRA
> b) historical trends in JRA and CMIP
> c) projected trends in CMIP
> d) applied perturbation

For c), trends for the period 2015-2100, or perhaps 2100-2080 mean minus 2015-2005 mean?

**Comment by adele157** [comment]
You can add figures to the summary page just the same as issues, I've added an example one in now:
[https://github.com/adele157/easterlies-collaborative-project/blob/master/Figure_outline.md](https://github.com/adele157/easterlies-collaborative-project/blob/master/Figure_outline.md)

This issue is good for discussion though, because we'll get email notifications and I'm guessing there will be lots of discussion to be had!

**Comment by adele157** [comment]
Should we decide what journal we're aiming for as well? i.e. Are we aiming for 4 or 10 figures?

**Comment by adele157** [comment]
I wonder if for the CMIP comparison, rather than focusing on the multi-model mean changes, we could perhaps focus on the scatter in the CMIP historical and future projections. ie. Make the argument that CMIP doesn't tell us much, because the models are all over the place and don't even reproduce the historical period well. So that's why we have chosen an idealised perturbation and simply amplified the existing wind pattern.

**Comment by matthew-england-unsw** [comment]
Yes agreed Adele - good call.

**Comment by julia-neme** [comment]
OK, how about then:
*Figure 1*
a), b) JRA55-do mean fields of uas and vas for 1958 - 2015
c), d) JRA55-do trends in uas and vas for 1958 - 2015
e), f) Boxplots of CMIP6 models trends in different regions, with JRA55-do as reference.

I suggest the boxplots are a nice way of showing the spread in trends and differences with JRA, since they are highly variable depending on location. Some regions are OK in CMIP6, some are really bad. I've uploaded the figure for you to see.

**Comment by matthew-england-unsw** [comment]
Lovely plot Julia!!   :-)
So this is historical wind trends?  What years?
I wonder about a final row showing (say) the multi-model mean wind anomalies [2081-2100] or trends [2000-2100] for CMIP6 (under the high emissions scenario)? 
Biases in CMIP6 models comes out nicely here.  
Love the circumpolar tendencies that can be clearly seen in the box plots as well.

**Comment by matthew-england-unsw** [comment]
> Should we decide what journal we're aiming for as well? i.e. Are we aiming for 4 or 10 figures?

I'd vote for 10 I think.  Unless we make 10 and decide that 4 tell the essential story, and then we can move the other 6 to Suppl. Info?

**Comment by StephenGriffies** [comment]
I find it easier to write the long manuscript first to be sure the story is solid and complete. Then, could cut back to shorter GRL size if warranted/desired.

**Comment by wghuneke** [comment]
I suggest we need figures that show (somewhat in this order):
- two-time scale response (e.g. using [SSH](https://github.com/adele157/easterlies-collaborative-project/issues/23)): quick Ekman response in first year, reversal of SSH signal within a couple of years (we think due to sea ice export increasing the sea ice/DSW formation)
- increase in DSW formation (e.g. [bottom age](https://github.com/adele157/easterlies-collaborative-project/issues/11), [bottom temp/salt](https://github.com/adele157/easterlies-collaborative-project/issues/4), map of surface water mass transformation rates)
- https://github.com/adele157/easterlies-collaborative-project/issues/10, increase in formation, [increase in export](https://github.com/adele157/easterlies-collaborative-project/issues/22)

**Comment by AndyHoggANU** [comment]
I've been thinking a bit about this figure order. I think we have two possibilities:

1. Something linear, where we build up a picture gradually -- starting with temp and salt evolution, SSH response, sea ice evolution and export, and culminating in the DSW and off-shelf age changes. (Note that we may need a second set of figures showing different experiments to distinguish between hypotheses.)
2. Show the headline result first (well, as figure 3, assuming a CMIP 6 winds + model configuration figure). This, I think, would be the DSW change/age change figure. Then, go back to investigating other quantities to disentangle hypotheses.

I quite like the second option for this paper, accepting that it is a little unusual. Any thoughts?

Also, I am not quite 100% convinced that we have fully disentangled the Ekman from the sea ice export hypothesis. Do we have a good experiment that could actually separate these two possibilities? That might be important for the storyline I am proposing...

**Comment by adele157** [comment]
I like this suggestion Andy, but I would be tempted to show the SSH time series before the DSW plot. That way we could start off by saying that we might naively to expect the easterlies to pile up water on the shelf, and indeed we do see that initially, but it is quickly overwhelmed by a larger opposing response by the DSW. Then go into DSW mechanisms. I agree I don’t think we’ve distangled the mechanisms completely yet, let’s discuss today what analysis / simulations we need to do that properly.

**Comment by matthew-england-unsw** [comment]
I think that’s a great idea Andy if the aim  is to make the paper headline about the DSW response.   But over the wider ocean there’s a largely coherent circumpolar (and counter-intuitive) response that is pretty impressive too, that remains when we pulled away the DSW formation region wind anomalies.  To me that’s the headline story, and the DSW changes are kind of secondary or at least consequent on that.  But that could just be me, DSW enthusiasts would be taken in by the storyline you propose.    :-)

**Comment by wghuneke** [comment]
Thanks for updating Fig 1 @julia-neme, looks good! J

ust a few minor things: Can you change the colorbar ticks (control) to a spacing of 1? Or maybe even 2 and for the anomaly 0.5 and then you can increase the font size for all the ticks and the other text. And maybe show a few less arrows, I find it looks slightly busy, but that might just be me.

**Comment by wghuneke** [comment]
Also adding a short summary on what we discussed today (mostly for @StephenGriffies ):

We continued our discussion about the paper outline, Adele and Andy updated the [Figure_outline.md](https://github.com/adele157/easterlies-collaborative-project/blob/master/Figure_outline.md) document accordingly. 

We have three hypotheses for different mechanisms that could explain the observed response: **A** local wind (katabatics) mechanism, **B** Upwelling mechanism, **C** Sea ice mechanism. We can rule out **A** with the additional perturbation experiment we did. The next steps are to do a bit more analysis/run additional simulations to disentangle **B** and **C**. Atm, we _think_ **C** is more relevant. 

Adele is currently running two more simulations for the UP case: a "meridional winds only" and a "zonal winds only" case, where only one component is perturbed and the other is kept as in the control run. The idea is that it will change the curl, hence Ekman upwelling, but not so much the sea ice advection.

**Comment by matthew-england-unsw** [comment]
Thanks @wghuneke great summary from today.  👍

**Comment by matthew-england-unsw** [comment]
PS:  @julia-neme given that the wind experiments are simply +10% and -10% south of the purple line, the 2 x anomaly plots on the right are maybe not needed, and so instead you could use the full 'real estate' of the page width to devote to a single circumpolar wind vector plot....?   I'm also curious how this looks without uniform vectors (which can be deceiving in the 'blank' low wind regions).   e.g. if taking up the full page width and using a proper vector scale, we will maybe see clearly the full wind field and then knowing the expts are +/- 10% anomalies on this mean field could be sufficient?

**Comment by adele157** [comment]
Yes agreed Matt, I think we could lose the 2 right panels. Julia, what do you think about using the model land/sea mask too, which cuts out some more of the Ross and Weddell?

**Comment by julia-neme** [comment]
Yeah! Will update based on all these comments :) Thanks!

**Comment by julia-neme** [comment]
# Fig 1 updated

Wind speed in contours, unscaled wind vectors, zero divide line in magenta, 1000m isobath in black, red boxes mark masked region for the additional experiment.

![fig1_experimentalsetup_v1-unscaled (1)](https://user-images.githubusercontent.com/51519847/134875009-e8dbc8e6-65ad-414d-ab3a-2d672a359916.jpg)

**Comment by wghuneke** [comment]
I started working on the bottom age plot (paper Fig 3) and created a list with items we need to agree upon:

- Which projection do we want to use? Stereo is nice, but I find it hard to see details on the continental slope. (compare my two version of the age plot)
- 1 km isobath: We should use the same definition
- Naming of experiments
- Which years do we want to show (if it's not a time series)? Average over the last year or last 5 years? (I found it weird to average the age over 5 years, but maybe that's ok.)

**Comment by wghuneke** [comment]
Here two version of the bottom age plot for the last year of the model run (year 15). The control shows the normalised (length of model simulation) age and the anomalies are the anomaly from the control (not normalised). 

![Fig3_Bottom_Age_stereo](https://user-images.githubusercontent.com/15355753/137081258-80e19578-36f3-4e09-818e-f78d0af66ccb.png)

![Fig3_Bottom_Age_cartesian](https://user-images.githubusercontent.com/15355753/137081279-5e1a4c01-7c32-4a4e-9699-33fde1ead8cc.png)

**Comment by matthew-england-unsw** [comment]
Nice Wilma!   I agree - polar-stereographic is not so ideal here, given our shelf focus.   So the bottom set gets my vote.  And I agree - age differences averaged over last 5 yrs is not as intuitive as age for last year, but I suppose either/or is fine and maybe what's important is to match up with whatever time window we're using to examine hydrographic property anomalies?

**Comment by matthew-england-unsw** [comment]
Experiment names?  How about:
![image](https://user-images.githubusercontent.com/47405285/137084743-e0b44d36-01eb-4f73-a413-7543658f0e9e.png)

**Comment by wghuneke** [comment]
Nice! Interesting that Adelie went up for the no katabatics case. The plot also shows how DSW formation changes less in the Ross Sea than, e.g., in the Weddell Sea.

**Comment by AndyHoggANU** [comment]
I have been working a little on Figure 5 -- which is designed to show the net buoyancy fluxes. I decided there is little purpose in showing maps, and that instead we should restrict ourselves to shelf-averaged or shelf-summed timeseries or climatologies.

Here (a) and (d) show the annual cycle of net heat flux and net FW flux averaged over the shelf, along with the up and down anomalies. One point here is that FW flux is almost entirely melt + river, the latter of which is constant, so for anomaly plots (e and f) I just sum the melt (to give a flux in Sv).

I decided that it may be worth including the zonal and meridional expts in the remaining panels, which show the anomaly in climatology for each experiment (b,e) and the 12-month running mean (c,f). If we retain these lines here, that may change the order we explain things in the paper...
![Figure5](https://user-images.githubusercontent.com/26753100/140189066-33ac861a-5758-482d-b5e2-1c460828ba38.png)
.

**Comment by wghuneke** [comment]
Update on naming and colours

CONTROL -> 'k'
WIND+ -> 'C0'
WIND$-$ -> 'C1'
WIND+_$zonal$ -> 'C2'
WIND+_$merid$ -> 'C3'
WIND+_$noDSW$ -> 'C4' (might change the naming)

**Comment by adele157** [comment]
Andy, a comment on Figure 5. I think the point here is to show that for the UP and DOWN simulations: a) heat flux changes are insignificant compared to freshwater flux changes, and b) the changes in sea ice formation (i.e. melt diagnostic) dominate the change in the freshwater flux change. I think we could make this point most clearly if all 3 of these things have the same units, and then could we show them all on the same plot? 

Probably also the changes in melt (i.e. ice export) could be kept for later when we use the zonal/meridional experiments to prove the sea ice mechanism?

**Comment by adele157** [comment]
I did some editing of the [Draft figure outline](https://github.com/adele157/easterlies-collaborative-project/blob/master/Figure_outline.md) page with a suggestion of when to introduce the zonal/meridional experiments. 

Basically, I think even before we introduce the new zonal/meridional experiments, we could first show the Ekman pumping Figure 6 and argue that the increase in upwelling is driven by zonal winds, and then the sea ice advection Figure 7 and argue that the sea ice advection is driven by meridional winds. So on these figures we wouldn't actually use the results from those experiments.

Then Figure 8 shows the SSH and DSW transport changes from the zonal/meridional experiments and after that we make the connection with the meridional winds driving the DSW changes.

Possibly this is actually what we said last meeting. But today a lot of the plots before Figure 8 already had the zonal/meridional experiments on them.

**Comment by wghuneke** [comment]
Here is a time series of sea ice volume (circumpolar and annual integral). The idea was to show this together with Andy's Fig5f): WIND+ and WIND+merid have more sea ice formation (neg melting), but sea ice volume is lower for these experiments. This means that there must be more sea ice export to the north. 

![Screenshot_SeaIceVolume_CircumpolarIntegral_AnnualIntegral](https://user-images.githubusercontent.com/15355753/140244039-6332cda4-aaf9-4b73-8f2c-58559fe2af8a.png)

**Comment by adele157** [comment]
I guess I was thinking rather than showing this as a separate plot, can we
combine this with the melt diagnostic to more accurately compute the sea
ice export off the shelf? Or at the very least can we use the changes in
sea ice volume over time here to quantify what the error is in assuming
that the change in formation minus melt (i.e. the melt diagnostic) = change
in export?

**Comment by AndyHoggANU** [comment]
Hi @adele157 - I've been working on the relative buoyancy fluxes and have struck a flaw in our logic. All the calculations I have done indicate that buoyancy fluxes for heat are greater than for salt. OK, so it's most likely I have made an error. But, at it's most basic, I can calculate `alpha*net_surface_heat_flux/C_p` and `beta*SSS*net_fw_flux` to be both in kg/s/m^2, and I get something like this (for the 15-year running-mean):
![Screen Shot 2021-11-05 at 2 46 27 pm](https://user-images.githubusercontent.com/26753100/140455283-b02b854d-0c22-4a0a-9118-b225fd402147.png)
Any thoughts?

**Comment by adele157** [comment]
Hmmm that does seem problematic. Where’s the code for this?

**Comment by AndyHoggANU** [comment]
It's a bit scratchy -- I didn't calculate local fluxes, I just got an estimate for alpha and beta using averages, and then converted to see what it looked like. See

https://github.com/adele157/easterlies-collaborative-project/blob/master/notebooks/buoyancy_fluxes/Heat-FW-Fluxes-Figure.ipynb

**Comment by StephenGriffies** [comment]
@AndyHoggANU did you also compute the contribution from salt fluxes?  I would presume that salt from the ice model is important too, as well as the SSS restoring.  

sfc_salt_flux_total  = sfc_salt_flux_restore + sfc_salt_flux_coupler

**Comment by adele157** [comment]
Yes, I think there's something wrong with your code @AndyHoggANU. Not sure exactly what, but when I use monthly alpha and beta, and also a slightly different shelf mask to yours, I find that the freshwater fluxes are ~5 times larger than the heat fluxes over the shelf, see here:
https://github.com/adele157/easterlies-collaborative-project/blob/Adele/notebooks/buoyancy_fluxes.ipynb

**Comment by AndyHoggANU** [comment]
Yep, right you are. I had forgotten to convert temperature to Celsius. Will fix and see what happens.

**Comment by AndyHoggANU** [comment]
OK, take 2 for Fig. 5
![Figure5(1)](https://user-images.githubusercontent.com/26753100/140597625-0886eef6-0fcd-4231-a2f5-19a332106f70.png)
.

**Comment by adele157** [comment]
Looks good Andy. Do we also want to show that the buoyancy flux anomaly is
dominated by sea ice? i.e. adding another line to the anomaly plot for melt
(in same buoyancy units)? Or another panel if it's too busy? I know early
on Matt kept wondering if the salinity changes could be driven by
evaporation, so might be nice to rule that out here.

**Comment by AndyHoggANU** [comment]
Yeah, I was thinking that they would just be too similar, but perhaps we can do it this way (I think it is self-explanatory, but I can explain if you want).
![Figure5-2](https://user-images.githubusercontent.com/26753100/140628956-57bc2eb9-ceb0-4525-8cf0-b0d46bc1a655.png)

**Comment by matthew-england-unsw** [comment]
Thanks Andy. I think if it’s that similar it can also just be noted in the caption / ms text.

**Comment by adele157** [comment]
Yes agreed, a comment like “sea ice processes are responsible for x% of the
change in freshwater flux” would be fine instead of showing on the plot.
Thanks for showing though Andy!

**Comment by AndyHoggANU** [comment]
Yep, that works. I will calculate the percentage.

**Comment by julia-neme** [comment]
First take at figure 8: 

![figure-8-v1](https://user-images.githubusercontent.com/51519847/140663299-a76fc84c-3c79-4a6d-93d7-307c4f898998.jpg)

It becomes really clear how the increase in DSW comes from the meridional component of the wind!!

**Comment by adele157** [comment]
Yeah, that's great! Does this have any smoothing applied to it yet? I would
argue for removing some of the seasonality on the DSW export lower plot.

**Comment by julia-neme** [comment]
That one has got 6 month rolling mean, I could do either 12 or annual averages??

**Comment by adele157** [comment]
Yeah, maybe give them a try, see what it looks like? Alternatively, we could show anomalies for the DSW export (I know we had a discussion on this on Thursday, can't remember if we decided that was a good or bad idea? @AndyHoggANU?), then the seasonality could be removed by subtracting the control climatology.

**Comment by julia-neme** [comment]
I think we had talk about showing the control as well? Anyways, here are the two options, 12 month running mean and anomalies from control:

![figure-8-v2](https://user-images.githubusercontent.com/51519847/140667855-dab498c1-1b00-40c2-9bd3-7fc7b8d0f651.jpg)
![figure-8-v4](https://user-images.githubusercontent.com/51519847/140667859-63e1ec67-2e75-43d8-a2a6-e98a409ed962.jpg)

**Comment by adele157** [comment]
I like the 12 month running mean.

**Comment by matthew-england-unsw** [comment]
Same!  :-)   PS:  interesting to see that the seasonal cycle in DSW anomalies in the WIND+zonal and WIND+meridional cases are ~6 months out of phase, and then approximately cancel out in the total WIND+ case.   Could indicate places where zonal winds are cross-shelf I guess.   But it's kinda curious that this would be seasonally opposite to the meridional case... ( a side issue of course)

**Comment by AndyHoggANU** [comment]
Yeah, I see the point, but maybe it is interesting to include the seasonal variation of DSW export -- partly because the meridional and zonal are out of sync!! ANyone else find that interesting?

**Comment by matthew-england-unsw** [comment]
Haha yes I also saw that and thought it was interesting, see my comment above.  :-)

**Comment by AndyHoggANU** [comment]
OK, if you look at the first draft of figure 8 (not plotted as an anomaly) you can see that what is happening here is that the control has a seasonal peak in summer; this peak is delayed in the zonal wind case and onset earlier in the meridional wind case. So, it might be interesting, but maybe it will be hard to explain succinctly ... so I am now leaning towards saying we should just focus on the 12-month running mean.

**Comment by PaulSpence** [comment]
Hi Folks, attempt at Fig 6 - Ekman driven upwelling change - using calculation of Ekman pumping binned into surface density bins.

![Screen Shot 2021-11-10 at 9 48 32 am](https://user-images.githubusercontent.com/6644956/141018446-7b87a1be-7bbb-4986-b551-e177f20214bb.png)

 I separated the area integration into on shelf, and within perturbation zone off shelf to reveal upwelling in CDW off shelf. The WIND+x and WIND+y are the zonal and meridional wind components of WIND+. Clearly WIND+x dominates the upwelling. It is difficult to isolate the upwelling of CDW using surface rho classes since the surface rho changes a lot seasonally and in spatially, as shown below.

![Screen Shot 2021-11-10 at 9 48 50 am](https://user-images.githubusercontent.com/6644956/141018988-7f21b251-ae47-4f57-b431-f39e5552387c.png)

**Comment by adele157** [comment]
Pretty plots Paul! Is something wrong with the WIND+y lines? I'm thinking it should at least be similar to the control (i.e. there should still be upwelling over the open ocean, just no more upwelling perhaps than the control).

Maybe a few north-south transects of temperature with these rho0 isopycnals overlaid would help in identifying on what isopycnal the warm CDW is located?

**Comment by PaulSpence** [comment]
Re: Is something wrong with the WIND+y lines? I'm thinking it should at least be similar to the control (i.e. there should still be upwelling over the open ocean, just no more upwelling perhaps than the control).

WIND+y is just the dx_tauyf component of WIND+, not the meridional wind perturbation expt. So WIND+x + WIND+y  =WIND+

**Comment by adele157** [comment]
Got it! Sorry!

**Comment by wghuneke** [comment]
New go on **Figure** 3 with bottom age and salinity anomalies for WIND+ and WIND-. Figure shows the symmetry between the up and down case and introduces the changes in DSW formation. 

I plot the annual average of year 10 (2159), could change the salinity to an average over the last 5 years. (?)
![Fig3_Bottom_Age_Salt_Anomaly_cartesian](https://user-images.githubusercontent.com/15355753/141043534-a29d9359-3cfb-4d93-add9-5848fb0710eb.png)

**Comment by wghuneke** [comment]
Question on **Figure 9** on temperature changes: What depth levels and which experiments do we want to show? The temperature response is not really part of the mechanisms we discuss in the paper. The motivation to include a temperature plot was more to inform the community about what all of that means for the temperature and melting...

Here is a go of bottom temperature for the WIND+ and WIND- cases only. Any suggestions?
![Fig9_Temp_anomaly_cartesian](https://user-images.githubusercontent.com/15355753/141043816-2322cb79-fb69-478e-9c47-a7811c2e2d72.png)

**Comment by adele157** [comment]
Looks good. Yeah, I reckon salinity averaged over the last 5 years is good. Perhaps could try enlarging both figures too (e.g. stretch vertical axis of Fig 3, and for Fig 9 put one underneath the other, so they're bigger?).

**Comment by PaulSpence** [comment]
Some updated Ekman/rho plots for the Amundsen, Denman and Wedell region are here:
https://github.com/adele157/easterlies-collaborative-project/issues/20#issuecomment-958479138

Perhaps rho>27.7 for CDW classification?

**Comment by matthew-england-unsw** [comment]
> Question on **Figure 9** on temperature changes: What depth levels and which experiments do we want to show? The temperature response is not really part of the mechanisms we discuss in the paper. The motivation to include a temperature plot was more to inform the community about what all of that means for the temperature and melting...
> 
> Here is a go of bottom temperature for the WIND+ and WIND- cases only. Any suggestions? ![Fig9_Temp_anomaly_cartesian](https://user-images.githubusercontent.com/15355753/141043816-2322cb79-fb69-478e-9c47-a7811c2e2d72.png)

Very nice Wilma!  I think surface and bottom temperature responses would be good to plot.  Like you said, not because it's core to the mechanism, more for community interest.  Not necessarily on same plot either; maybe we want T and S changes for both WIND+ and WIND- on same plot.  So one Fig. for surface, one for bottom.

**Comment by adele157** [comment]
Haha Matt, I was literally writing this at the same time. :) 

For the bottom temperature plot, how well can we explain these changes? 

The warming regions I think can be easily linked to the changing vertical heat fluxes associated with the change in the lower overturning cell. i.e. Increased winds -> enhancement of lower cell -> warming in upper ocean near and downstream of DSW production regions.

For the cooling (i.e West Antarctica and much of eastern East Antarctica), I'm less sure of the mechanism. I think it may just be that the isopycnals are being pushed down against the coast. It would be interesting to see if this cooling is coming from the zonal, not meridional component of the change. If it is a heaving signal, do we want to show that by plotting a transect somewhere, or calculating the heave component of the temperature change?

**Comment by wghuneke** [comment]
I prepared the temperature anomaly (at the surface and bottom) figure for WIND+ and WIND-:
- I'm not sure if we need both experiments as there are only very few places where the response is not symmetric (and we show the symmetry throughout the paper, e.g. Fig 3)
- Maybe we can replace the WIND- panels with some figure that explains the cooling response (see Adele's comment above). A transect and/or a map of the depth change of some representative isotherm?
- The bottom salinity is at the moment already shown in Fig 3 which is why I didn't prepared the same figure for salinity. And the surface salinity would be a mirror of the sea ice plot, so maybe not that informative?

![Fig9_Temp_surface_bottom_anomaly](https://user-images.githubusercontent.com/15355753/141411604-0f2a7101-8bc4-417e-a728-569c8f3be1f3.png)

**Comment by AndyHoggANU** [comment]
Here is a revised Fig 8 that I have sent to Julia to review. Others might have comments. If yes, please post them here. Or comment in the pull request. Whatever ...
![Unknown-4](https://user-images.githubusercontent.com/26753100/142783193-d3298c2c-0d4f-491f-b83d-42ba60dbc175.png)

**Comment by julia-neme** [comment]
Looks great Andy, thank you! I think we had decided to show 12 month rolling mean though?

**Comment by AndyHoggANU** [comment]
Oh yeah, you’re right. Well, the code is there now — I just didn’t have time to extend to the other 3 potential versions. Did you want to do that, or should I?

**Comment by PaulSpence** [comment]
Sea Ice Figure of velocity and thickness anomalies:
![Screen Shot 2021-11-22 at 6 02 22 pm](https://user-images.githubusercontent.com/6644956/142819009-47448e99-539c-4b49-98a8-ed655595cf80.png)

Sea Ice Figure of ice  ice transport (volume*velocity) and thickness anomalies:
![Screen Shot 2021-11-22 at 6 12 15 pm](https://user-images.githubusercontent.com/6644956/142819127-5136a274-9a67-4df3-a4ec-792f83ab2f1b.png)

I think the velocity offers more clarity than the transport.

**Comment by adele157** [comment]
Yes the transport’s pretty small
In the Ross. Thanks for checking! Looks like you got the velocities much
smoother too, nice!
